dotnetv3/Glue/README.md

```diff
@@ -13,6 +13,10 @@ AWS Glue is a serverless data integration service that makes it easy to discover
 
 ## Code examples
 
+### Get started
```

Comment:
As we discussed, need to add instructions so the user can run the example here: for example, setting up the configuration files and for running the CDK and uploading the file needed.

Reply:
You could also generate this using Writeme templates, so you would get the more detailed scenario description from the metadata.

Reply:
Added instructions.